apps/roam/src/components/settings/NodeConfig.tsx

```diff
@@ -160,6 +169,53 @@ const NodeConfig = ({
             </div>
           }
         />
+        <Tab
+          id="github"
+          title="GitHub"
+          panel={
+            <div className="flex max-h-[70vh] flex-col gap-4 overflow-y-auto p-1">
+              <div className="mb-4 rounded bg-gray-50 p-4">
+                <p className="mb-2 text-sm text-gray-600">
+                  GitHub integration allows you to sync {node.text} pages with
+                  GitHub Issues. When enabled, you can:
+                </p>
+                <ul className="list-disc pl-5 text-sm text-gray-600">
+                  <li>Send pages to GitHub as issues</li>
+                  <li>Import and sync comments from GitHub</li>
+                  <li>Configure where comments appear in the page</li>
+                </ul>
+              </div>
+              <FlagPanel
+                title="GitHub Sync"
+                description={`When enabled, ${node.text} pages can be synced with GitHub Issues.`}
+                order={0}
+                parentUid={node.type}
+                uid={githubSyncUid}
+                value={node.githubSync}
+                options={{
+                  onChange: (checked) => setIsGithubSyncEnabled(checked),
+                }}
+              />
+
+              {isGithubSyncEnabled && (
+                <div className="mt-4 border-t pt-4">
```

Comment:
Let's remove this whitespace space
![image](https://github.com/user-attachments/assets/312d3436-671b-411e-ad8a-c01789a2ce37)